### PR TITLE
content nits

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: docs/welcome-to-libra-v2.md [hunk 1]
```diff
@@ -4,12 +4,16 @@ sidebar_label: Home
...
 
-<p>The Libra payment system is built on a secure, scalable, and reliable blockchain. It is backed by a reserve of high-quality liquid assets comprising cash or ca​sh eq​uivalents and very short-term government secu​rities. This will help ensure that people and businesses have confidence that their Libra Coins can be converted into their local currency. It is governed by the [Libra Association](link) and its subsidiary Libra Networks, tasked with developing and operating the Libra network and the Libra project.</p>
+The Libra payment system is built on a secure, scalable, and reliable blockchain. It is backed by a reserve of high-quality liquid assets comprising cash or ca​sh eq​uivalents and very short-term government secu​rities. This will help ensure that people and businesses have confidence that their Libra Coins can be converted into their local currency. It is governed by the [Libra Association](http://libra.org/) and its subsidiary Libra Networks, tasked with developing and operating the Libra network and the Libra project.
 
 <div className="margin-vert--xl" />
 
+<h2 id="test">Yo yo</h2>
```
Comment (jjwill) [inline docs/welcome-to-libra-v2.md]: Remove

File: docs/welcome-to-libra-v2.md [hunk 2]
```diff
@@ -4,12 +4,16 @@ sidebar_label: Home
...
+
+## ahoy matey
```
Comment (jjwill) [inline docs/welcome-to-libra-v2.md]: Remove

**Review by jjwill — APPROVED** [review]
LGTM

File: docs/welcome-to-libra-v2.md (hunk 2 above)
Comment (oriooctopus) [reply docs/welcome-to-libra-v2.md]: OH whoops. Sorry about that!

**Review by jjwill — APPROVED** [review]
LGTM

**Pull request merged**